nevooronni/Politico_web_app
# #163600854 An admin user can create a political party

#### What does this PR do?
This is a PR for a feature that allows a an admin user to create a political party

#### Description of Task to be completed?
The following are the tasks to be completed
* add html for this page
* style html markup with css
* Add javascript for modal functionality

#### How to manually test feature?
1. clone this repo
``https://github.com/nevooronni/Politico-Andela.git```

2. cd in folder
```cd Politico-Andela```

3 switch to feature branch 
```git checkout ft-create-political-party-163600854```

4. Open UI/html/index.html in your browser

5. Click the signup button on landing page
6. Should be able to view signup page where you will see a signup up form
7. On the bottom of the form click on the signin link
8. You should see a signin page click on the sigin now link to be taken to admin page
9. Click on the create political party button to see a modal form which you use to submit data

#### Pivotal Tracker story associated with this PR:
story id: #163600854

#### Screenshots
![screenshot from 2019-02-01 03-47-19](https://user-images.githubusercontent.com/20374625/52095675-fda11e80-25d4-11e9-8ace-5a0e61a49968.png)

![screenshot from 2019-02-01 03-47-30](https://user-images.githubusercontent.com/20374625/52095686-07c31d00-25d5-11e9-8d63-f68c89f67ddc.png)
